modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc

```diff
@@ -56,46 +56,91 @@ rpk redpanda start [flags]
 |*Value* |*Type* |*Description*
 
 |--advertise-kafka-addr |strings |A comma-separated list of Kafka
-addresses to advertise (scheme://host:port\|name).
+addresses to advertise.
+
+[,bash]
+----
+<scheme>:<host>:<port>\|<name>
+----
 
 |--advertise-pandaproxy-addr |strings |A comma-separated list of
-Pandaproxy addresses to advertise (scheme://host:port\|name).
+Pandaproxy addresses to advertise.
+
+[,bash]
+----
+<scheme>:<host>:<port>\|<name>
+----
+
+|--advertise-rpc-addr |string |The advertised RPC address 
 
-|--advertise-rpc-addr |string |The advertised RPC address (host:port).
+[,bash]
+----
+<host>:<port>
+----
 
 |--check |- |When set to false will disable system checking before
-starting redpanda (default true).
+starting `redpanda` (default true).
```

Comment:
```suggestion
starting `redpanda` (default `true`).
```
Not sure if redpanda should be in backticks, seems like we could just use "Redpanda"?

Reply:
used backticks as it's the redpanda process